src/demo-text-input/demo-text-input.css

```diff
@@ -1,27 +1,27 @@
 .demo-text-input {
-    border-top: 1px solid #ddd;
-    border-bottom: 1px solid #ddd;
-    padding: 1rem 0;
-    margin: 1rem 0;
+	border-top: 1px solid #ddd;
+	border-bottom: 1px solid #ddd;
+	padding: 1rem 0;
+	margin: 1rem 0;
 }
 
 .demo-text-input .container {
-    display: grid;
-    column-gap: 1rem;
-    grid-template-columns: 0.5fr 1fr 0.5fr min-content;
+	display: grid;
+	column-gap: 1rem;
+	grid-template-columns: 0.5fr 1fr 0.5fr min-content;
 }
 
 .demo-text-input .container > div:last-child {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
 }
 
 .demo-text-input .components-range-control__root {
-    flex-direction: row-reverse;
+	flex-direction: row-reverse;
 }
 
 .demo-text-input .components-range-control__number {
-    margin-left: unset !important;
-    margin-right: 1rem;
-}
```

Comment:
It seem to conflict with PHP lint rule where new line at the end of file is required.

Reply:
There's a new line at the end of each of the CSS files. When you run the linter, does it remove or not add new lines at the end?

Reply:
Sorry my bad. diff caused the confusion. 
I can confirm that it ads additional line at the end of the file.

Reply:
No worries! I wasn't sure if my editor settings were adding the new line or if it was the settings from the repo. My editor settings kept tripping me up before with this 😅